simple_network_sim/network_of_populations/__init__.py

```diff
@@ -42,20 +43,31 @@ class NetworkOfPopulation(NamedTuple):
     infectionProb: Dict[Time, float]
     initialInfections: Dict[NodeName, Dict[Age, float]]
     trials: int
+    start_date: dt.date
+    end_date: dt.date
```

Comment:
I think we are using camel case in this particular module.

Reply:
Yes you are right, I fixed it. I also added tests for the dateRange function which I had forgotten